libraries/azure_backend.rb

```diff
@@ -690,7 +688,7 @@ def respond_to_missing?(*several_variants)
   end
 
   def to_s
-    "Property is missing! The following properties are available: #{item.keys.map(&:to_s)}"
+    "#{type}/#{name} has following properties: #{item.keys.map(&:to_s)}."
```

Comment:
This should probably be: `has the following`

Reply:
Updated.